app/controllers/api/v2/people_actions.rb

```diff
@@ -47,16 +47,38 @@ def person_params
     end
 
     def person_attributes
-      person_params[:attributes].merge(ethnicity: ethnicity, gender: gender)
+      attributes = {}
+      attributes.merge!(person_params.fetch(:attributes, {}))
```

Comment:
Simplified this to handle the case where we don't have any attributes (and only relationships or neither) to patch.